ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -328,3 +328,10 @@ def fmin(self, y, name=None):
     def minimum(self, y, name=None):
         y_ivy = _to_ivy_array(y)
         return ivy.minimum(self._ivy_array, y_ivy)
+
+    @with_supported_dtypes(
+        {"2.5.0 and below": ("float32", "float64", "bool", "int8", "int32", "int64")},
+        "paddle",
+    )

```

Comment:
this is not necessary since this function supports all dtypes

Reply:
the decorator has been removed, also fixed some conflicts. please re-review the changes